### PR TITLE
Disable sentry in dev

### PR DESCRIPTION
**Pull request by mbarton**
Only catch and send JS errors to sentry if we are running in CODE or PROD

cc @akash1810 @Reettaphant 
Fixes #224

### CODE REVIEW

File: public/video-ui/src/app.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ const store = configureStore();
 const config = extractConfigFromPage();
 
 // publish uncaught errors to sentry.io
-Raven.config(config.ravenUrl).install();
+if(config.stage !== 'DEV')
```
Comment (akash1810) [inline public/video-ui/src/app.js]: Can we limit to PROD only as we sometime release feature branches to CODE.

Comment (mbarton) [inline public/video-ui/src/app.js]: Done

File: public/video-ui/src/app.js [hunk 2]
```diff
@@ -29,7 +29,8 @@ const store = configureStore();
 const config = extractConfigFromPage();
 
 // publish uncaught errors to sentry.io
-Raven.config(config.ravenUrl).install();
+if(config.stage !== 'PROD')
```
Comment (akash1810) [inline public/video-ui/src/app.js]: should this be `=== 'PROD'` ?

Comment (mbarton) [inline public/video-ui/src/app.js]: Oops yes it should

**Pull request merged**